### DISCUSSION TITLE
Successful Dual-Boot Thank You and Basic Questions

### DISCUSSION THREAD

**Discussion by VolvoBirkenstock** [mocaccinoOS/mocaccino#94] [General] [answered]
Hello, Mocaccino Gurus.  

I have managed to successfully dual-boot a formerly Sabayon/Windows installation into Mocaccino/Windows and have verified proper operation.  A HUGE thank you!!!  :-)  Here come the questions...

I've been trying to learn how to use luet and read https://www.mocaccino.org/docs/desktop/usage/ .  The link to Luet docs on that web page (https://luet-lab.github.io/docs/docs/concepts/overview/managing_packages/) fails.  
1. Where can I learn about all the luet commands? (Are "luet install", "luet uninstall", "luet upgrade" and "luet search" all of the commands supported by luet?)  
2. Where can I learn about all the packages that can be installed using luet? 
3. Where can I learn the descriptions of the contents of each package that luet can install?  

For example, if I want to install gkrellm, if the package that installs gkrellm is available, how do I find that package name? 
If a package that installs gkrellm is not available, how would _you_ install gkrellm on a Mocaccino system? 

Again, a HUGE thank you !!!

**Comment by alvintseng** [comment]
Hi VolvoBirkenstock
You can try this webpage-->https://luet.io/docs/
I only find this page for reference.

**Comment by joostruis** [comment, marked as answer]
I think you need to know what you are looking for. If there is anything missing for you, you can request it here: https://github.com/mocaccinoOS/community-repository/issues

**Reply by VolvoBirkenstock** [reply]
I would prefer to be able to know enough to seek and find, thereby reducing your burden.  However, I will provide requests in the location you provided. 

I tried to follow the [https://luet.io/docs/](https://luet.io/docs/) down into some examples, but I think the examples are suffering from "link rot".  Here's the links that don't work:
[https://github.com/mocaccinoOS/mocaccino-musl-universe/tree/master/multi-arch/packages/entities](https://github.com/mocaccinoOS/mocaccino-musl-universe/tree/master/multi-arch/packages/entities)
[https://github.com/mocaccinoOS/portage-tree/tree/master/multi-arch/packages/groups](https://github.com/mocaccinoOS/portage-tree/tree/master/multi-arch/packages/groups)
[https://github.com/mocaccinoOS/mocaccino-musl-universe/tree/master/multi-arch/packages/X](https://github.com/mocaccinoOS/mocaccino-musl-universe/tree/master/multi-arch/packages/X)
Here's the locations where I believe they might now best resolve: 
[https://github.com/mocaccinoOS/mocaccino-musl-universe/tree/master/](https://github.com/mocaccinoOS/mocaccino-musl-universe/tree/master/)
[https://github.com/mocaccinoOS/portage-tree/tree/master/](https://github.com/mocaccinoOS/portage-tree/tree/master/)
Please do not consider this a complaint.  It isn't.  When there are many moving parts and the load is heavy, it is difficult keeping everything perfect.
